libs/gi/ui/src/context/BuildEditContext.tsx

```diff
@@ -0,0 +1,12 @@
+'use client'
+import { createContext } from 'react'
+
+export type BuildEditContextObj = {
+  buildToEdit: string
+  //setBuildIdToEdit: (buildId: string) => void
+}
+
+export const BuildEditContext = createContext({
```

Comment:
could simplify this from an obj to an string, and maybe make "equipped" as default?
```suggestion
export const BuildEditContext = createContext<string | "equipped">("equipped")
```

Reply:
done